apps/web/public/static/locales/en/common.json

```diff
@@ -2699,6 +2699,9 @@
   "confirm_reassign_unavailable": "Manual Reassign to unavailable host",
   "reassign_unavailable_team_member_description": "Are you sure you want to reassign this booking to an unavailable host?",
   "yes_reassign": "Yes, reassign",
+  "attendee_timezone_info": "Attendee Timezone",
+  "event_time_in_attendee_timezone_info": "Event Start Time In Attendee Timezone",
+  "event_end_time_in_attendee_timezone_info": "Event End Time In Attendee Timezone",
```

Comment:
we should add translations for the variable itself too:

attendee_timezone_variable: "Attendee timezone"
event_time_in_attendee_timezone_variable: "Event time in attendee timezone" 
event_end_time_in_attendee_timezone_variable: "Event end time in attendee timezone"